system-variables.md

```diff
@@ -1074,3 +1074,9 @@ This variable is an alias for _transaction_isolation_.
 - Scope: SESSION | GLOBAL
 - Default value: ON
 - This variable controls whether to use the high precision mode when computing the window functions.
+
+### tidb_enable_engine_fallback
+
+- Scope: SESSION | GLOBAL
+- Default value: OFF
+- This variable controls whether to retry execution with another engine when the used engine's failure causes execution failure of SQL statement. If the variable turns on, currently TiDB supports retrying execution with TiKV when TiFlash's failure causes execution failure of SQL statement.
```

Comment:
The options in this page are in alphabetical order (the order which they were added to the server is not useful to end-users). Please move it to the correct place :-)